code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -810,12 +783,17 @@
 	illustration = "writing_syndie"
 
 /obj/item/storage/box/syndicate/contractor_loadout/PopulateContents()
+	new /obj/item/mod/control/pre_equipped/infiltrator(src)
 	new /obj/item/clothing/head/helmet/space/syndicate/contract(src)
 	new /obj/item/clothing/suit/space/syndicate/contract(src)
 	new /obj/item/clothing/under/chameleon(src)
 	new /obj/item/clothing/mask/chameleon(src)
-	new /obj/item/storage/fancy/cigarettes/cigpack_syndicate(src)
 	new /obj/item/card/id/advanced/chameleon(src)
+	new /obj/item/computer_disk/syndicate/camera_app(src)
```

Comment:
It would be better to just include this as an application on the PDA that they're given to do their tasks on.

Reply:
Okay